portlets/src/main/webapp/vuesLocales/locale_en.json

```diff
@@ -18,11 +18,15 @@
   "functionalConfiguration.table.cancel": "Cancel",
   "functionalConfiguration.table.save": "Save",
   "functionalConfiguration.table.empty": "No data",
+  "functionalConfiguration.activeTermsAndConditions": "Active user terms and conditions",
+  "functionalConfiguration.termsAndConditionsWebContentUrl": "Web content url",
+  "functionalConfiguration.termsAndConditionsWebContentUrl.info": "Webcontent link should be into collaboration shared folder (eg:/myTerms.pdf). This document should have a version."
   "functionalConfiguration.table.group": "Group: ",
   "functionalConfiguration.table.order": "Order: ",
   "functionalConfiguration.highlightSpaces.edit.portletName": "Portlet name: ",
   "functionalConfiguration.highlightSpaces.edit.groupSelection": "Group selection: ",
   "functionalConfiguration.highlightSpaces.edit.group": "Group",
   "functionalConfiguration.highlightSpaces.edit.close": "Close",
-  "functionalConfiguration.highlightSpaces.edit.save": "Save"
+  "functionalConfiguration.highlightSpaces.edit.save": "Save",
+  "functionalConfiguration.termsAndConditions.fileNotFound": "Le fichier est introuvable"
```

Comment:
This string is only for the server log message ? If so, no need to translate them, server logs message should always be in english.